src/Integration.Vsix/Telemetry/TelemetryLogger.cs

```diff
@@ -74,49 +71,5 @@ public void ReportEvent(TelemetryEvent telemetryEvent)
                     break;
             }
         }
-
-        int IOleCommandTarget.Exec(ref Guid pguidCmdGroup, uint nCmdID, uint nCmdexecopt, IntPtr pvaIn, IntPtr pvaOut)
```

Comment:
Duncan investigated and this is not being called, only for known IOleCommandTargets (like the package). This not being called means the command that was raised has no handler (since the package was not loaded) and we loose a piece of info that might be useful (in case it's not at this point) - the package and the version handling the command.